TestTools/StreamChatTestMockServer/Robots/ParticipantRobot.swift

```diff
@@ -201,13 +201,34 @@ public class ParticipantRobot {
         )
         return self
     }
+    
+    @discardableResult
+    public func replyToMessage(_ text: String, withAttachment: Bool, toLastMessage: Bool = true) -> Self {
```

Comment:
quoteMessage or quoteReplyMessage. We can't name `reply` only because, we can confuse it with `threadReply` 
```suggestion
    public func quoteMessage(_ text: String, withAttachment: Bool, toLastMessage: Bool = true) -> Self {
```